plugins/repository-azure/src/main/java/org/elasticsearch/repositories/azure/AzureBlobContainer.java

```diff
@@ -156,7 +156,8 @@ public void move(String sourceBlobName, String targetBlobName) throws IOExceptio
 
             blobStore.moveBlob(blobStore.container(), source, target);
         } catch (URISyntaxException | StorageException e) {
-            logger.warn("can not move blob [{}] to [{}] in container {{}}: {}", sourceBlobName, targetBlobName, blobStore.container(), e.getMessage());
+            logger.warn("can not move blob [{}] to [{}] in container {{}}: {}", sourceBlobName, targetBlobName, blobStore.container(),
```

Comment:
Did this file happen to be mentioned in the checkstyle suppressions? If so can you remove it now?

Reply:
Indeed. Did not know about this file TBH :)
I removed all those with 019b431